### PR TITLE
[RTM] ENH: Use --template flag to specify normalization target

### PR DESCRIPTION
**Pull request by effigies**
This PR separates the normalization directive (`--output-space template`) from the specification of a normalization target via `--template <TEMPLATE>`.

Currently only `MNI152NLin2009cAsym` is supported, but this lays the groundwork for additional targets with minimally intrusive modifications.

See discussion in #487.

Depends on poldracklab/niworkflows#158.

### CODE REVIEW

File: fmriprep/workflows/epi.py [hunk 1]
```diff
@@ -636,7 +638,9 @@ def _aslist(in_value):
         return [in_value]
 
     gen_ref = pe.Node(GenerateSamplingReference(), name='GenNewMNIReference')
-    gen_ref.inputs.fixed_image = op.join(get_mni_icbm152_nlin_asym_09c(), '1mm_T1.nii.gz')
+    template_str = {'MNI152NLin2009cAsym': 'mni_icbm152_nlin_asym_09c'}[template]
```
Comment (chrisfilo) [inline fmriprep/workflows/epi.py]: Could we move this dictionary into one central place? it is being used a couple times

Comment (effigies) [inline fmriprep/workflows/epi.py]: Was thinking it might make sense in `niworkflows.data`.

Comment (chrisfilo) [inline fmriprep/workflows/epi.py]: yeah that makes sense

File: fmriprep/cli/run.py [hunk 2]
```diff
@@ -79,10 +79,13 @@ def get_parser():
                              'default to compensate for field inhomogeneities.')
     g_conf.add_argument(
         '--output-space', required=False, action='store',
-        choices=['T1w', 'MNI152NLin2009cAsym',
-                 'fsnative', 'fsaverage', 'fsaverage6', 'fsaverage5'],
-        nargs='+', default=['MNI152NLin2009cAsym', 'fsaverage5'],
+        choices=['T1w', 'template', 'fsnative', 'fsaverage', 'fsaverage6', 'fsaverage5'],
+        nargs='+', default=['template', 'fsaverage5'],
         help='volume and surface spaces to resample functional series into')
```
Comment (chrisfilo) [inline fmriprep/cli/run.py]: Sorry I did not catch this before - could you briefly describe what 'template' means in this help string and refer people to `--template`?

**Comment by effigies** [comment]
Updated the help string. Want to make sure this looks okay in the docs before merging.

**Comment by effigies** [comment]
Fancy docs:

fancy_docs

**Comment by chrisfilo** [comment]
Could you rebase/merge master?

**Comment by effigies** [comment]
Rebased.

**Pull request merged**